src/lambda/templates/lambdaTemplates.ts

```diff
@@ -43,12 +43,23 @@ export class LambdaTemplates {
     `
     public static readonly GET_POLICY_TEMPLATE = `
     <h1>
-        Policy for <%= FunctionName %>...
+        Policy for <%= FunctionName %> ...
     </h1>
     <p>Policy:
         <pre><%- JSON.stringify(JSON.parse(Policy), null, 4) %></pre>
     </p>
     `
+    public static readonly GET_POLICY_TEMPLATE_ERROR = `
```

Comment:
I'm assuming that lodash handles HTML sanitation here, is that correct?

Reply:
No. We'll want to use `_.escape(text-to-sanitize)` where needed.

Reply:
We should probably use that for all of the strings in this template. Better safe than sorry.

Reply:
I've made #160, since there are more templates than the scope of changes in this PR